### PR TITLE
ExtraHop: Feature - PAPP-26197: Python conversion

### PR DESCRIPTION
**Pull request by ighelani-crest**
#### Please check if your PR fulfills the following requirements:
- [ ] Testing of all the changes has been performed (for bug fixes / features)
- [ ] The readme.html has been reviewed and added / updated if needed (for bug fixes / features)
- [x] Use the following format for the PR description: `<App Name>: <PR Type> - <PR Description>`
- [ ] Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.
- [ ] Verify all checks are passing.
- [ ] Do NOT use the `next` branch of the forked repo. Create separate feature branch for raising the PR.
- [ ] Do NOT submit updates to dependencies unless it fixes an issue.
#### Please check the type of change your PR introduces:
- [ ] New App
- [ ] Bugfix
- [x] Feature (Python3 conversion)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Documentation
- [] Other (please describe): 

## Security Considerations (REQUIRED)
- [ ] If you are exposing any endpoints using a [REST handler](https://docs.splunk.com/Documentation/SOAR/current/DevelopApps/RESTHandlers), 
  please document them in the `readme.html`.
- [ ] If this is a new connector or you are adding new actions
    - [ ] Please document in the `readme.html` all methods (eg, OAuth) used to authenticate 
      with the service that the connector is integrating with.
    - [ ] If any actions are unable to run on SOAR Cloud, please document this in the `readme.html`.
- [ ] Are you introducing any new cryptography modules? If yes, please elaborate their purpose:
- [ ] Are you are accessing the file system? If yes, please verify that you are only accessing paths returned through 
the [Vault](https://docs.splunk.com/Documentation/SOARonprem/5.2.1/DevelopApps/AppDevAPIRef#Vault) API.
- [ ] Are you are marking code to be ignored by Semgrep with [`nosemgrep`](https://semgrep.dev/docs/ignoring-files-folders-code/#ignoring-code-through-nosemgrep)?
If yes, please provide justification in an additional comment next to the ignored code.

  

## Release Notes (REQUIRED)
- Provide release notes as part of the PR submission which describe high level points about the changes for the upcoming GA release.

### CODE REVIEW

**Comment by dhwanis-crest** [comment]
The MR will be merged once all the pipeline passes and all the dependencies are merged (asset and playbooks) and the pipeline passes successfully.

File: readme.html [hunk 1]
```diff
@@ -1,9 +1,23 @@
...
-	<p>Extrahop and Phantom connect through simple, powerful REST APIs, making it easy to build and iterate new use cases to get the most value for the least effort, a vital capability for thinly stretched enterprise security teams.</p>
+	<p>For ExtraHop installation and configuration instructions visit <a href="https://www.extrahop.com/customers/community/bundles/extrahop-created/">bundles.extrahop.com</a>.</p>
+	<p>The Splunk SOAR integration for ExtraHop enables you to automate and orchestrate rapid security investigation, response, and remediation workflows. ExtraHop Reveal(x) provides a uniquely rich, real-time data source by turning unstructured packets into structured wire data and analyzing it in real-time. Based on this data, you can confidently configure Splunk SOAR to automate security workflows and investigations and orchestrate precise, rapid responses to security threats more effectively than ever before.</p>
+	<p>The Extrahop and Splunk SOAR connect through simple, powerful REST APIs, making it easy to build and iterate new use cases to get the most value for the least effort, a vital capability for thinly stretched enterprise security teams.</p>
```
Comment (splunk-jessica) [inline readme.html:21]:
```suggestion
	<p>Extrahop and Splunk SOAR connect through simple, powerful REST APIs, making it easy to build and iterate new use cases to get the most value for the least effort, a vital capability for thinly stretched enterprise security teams.</p>
</body>
```

File: extrahop.json [hunk 2]
```diff
@@ -345,34 +414,29 @@
         {
             "action": "get peers",
             "identifier": "get_peers",
-            "description": "Get a list of peers that a device communicated with in the last N minutes",
-            "verbose": "This action retrieves a list of all of the peers that a device communicated with in the last N minutes, optionally filtered by role and/or protocol.",
+            "description": "Get a list of peers that a device communicated within the last N minutes",
```
Comment (splunk-jessica) [inline extrahop.json:417]:
```suggestion
            "description": "Get a list of peers that a device communicated with within the last N minutes",
```

File: extrahop.json [hunk 3]
```diff
@@ -345,34 +414,29 @@
...
+            "verbose": "This action retrieves a list of all of the peers that a device communicated within the last N minutes, optionally filtered by role and/or protocol. Either 'ip' or 'eh_api_id' parameter is required. If both the parameters are provided, 'eh_api_id' will be considered.",
```
Comment (splunk-jessica) [inline extrahop.json:418]:
```suggestion
            "verbose": "This action retrieves a list of all of the peers that a device communicated with within the last N minutes, optionally filtered by role and/or protocol. Either 'ip' or 'eh_api_id' parameter is required. If both the parameters are provided, 'eh_api_id' will be considered.",
 ```

Comment (dhwanis-crest) [reply extrahop.json]: Done

File: extrahop.json (hunk 2 above)
Comment (dhwanis-crest) [reply extrahop.json]: Done

File: readme.html (hunk 1 above)
Comment (dhwanis-crest) [reply readme.html]: Done

**Comment by dhwanis-crest** [comment]
The sanity test pipeline fails because it expects https://us-west-2.console.aws.amazon.com/codesuite/codebuild/875003031410/projects/sanity-tests/batch/sanity-tests%3Ac2c60291-2133-4db9-a366-bc42b433c933/env_var?region=us-west-2 the pytest of the app in main branch (which is not yet merged in main) whereas the integration test pipeline passes as it is able to find pytest with the same name as app branch name - https://us-west-2.console.aws.amazon.com/codesuite/codebuild/875003031410/projects/app-integration-tests/build/app-integration-tests%3A15855e86-69c9-4107-85e6-8bacbb288505/env_var?region=us-west-2

File: extrahop.json [hunk 4]
```diff
@@ -57,110 +61,138 @@
             "read_only": true,
             "parameters": {
                 "ip": {
-                    "description": "IP Address",
+                    "description": "Comma separated IP addresses",
```
Comment (tonyc-phantom) [inline extrahop.json:64]: This should be 'Comma-separated'

**Review by tonyc-phantom — CHANGES_REQUESTED** [review]

File: extrahop.json (hunk 4 above)
Comment (dhwanis-crest) [reply extrahop.json]: Done

**Review by tonyc-phantom — APPROVED** [review]

**Review by splunk-jessica — APPROVED** [review]

**Pull request merged**